docs/reference/configuration/queue.rst

```diff
@@ -0,0 +1,479 @@
+.. _queue-system-chapter:
+
+Queue system
+============
+
+The queue system is responsible for scheduling, running, monitoring and acting
+on a set of realizations.
+
+A number of queue systems are pre-installed with ERT:
+
+- ``LOCAL`` — run all computations locally, on your machine. :ref:`Details.<local-queue>`
+- ``LSF`` — send computation to an LSF cluster. :ref:`Details.<lsf-systems>`
+- ``TORQUE`` — send computation to a TORQUE or PBS cluster. :ref:`Details.<pbs-systems>`
+- ``SLURM`` — send computation to a Slurm cluster. :ref:`Details.<slurm-systems>`
+
+Select the system using the ``QUEUE_SYSTEM`` keyword. For example, the
+following line in your ERT configuration file specifies that ERT should use the
+``LOCAL`` system::
+
+  QUEUE_SYSTEM LOCAL
+
+This page documents the configuration options available for each queue system.
+Some of the options apply to all systems, others only to specific queue systems.
+
+
+.. _all-queues:
+
+Options that affect all queue systems
+-------------------------------------
+
+In addition to the queue-specific settings, the following options affect
+all queue systems. These are documented in :ref:`ert_kw_full_doc`.
+
+* ``MAX_SUBMIT`` — see :ref:`List of keywords<max_submit>`
+* ``MAX_RUNTIME`` — see :ref:`List of keywords<max_runtime>`
+* ``STOP_LONG_RUNNING`` — see :ref:`List of keywords<stop_long_running>`
+* ``JOBNAME`` — see :ref:`List of keywords<jobname>`
+* ``NUM_CPU`` — see :ref:`List of keywords<num_cpu>`
+
+
+.. _local-queue:
+
+LOCAL queue
+-----------
+
+Let's create ``local_queue.ert`` with the following content:
+
+.. literalinclude:: ../queue/local_queue.ert
+
+In addition to this config, we'll also need a job config ``queue_test_job``:
```

Comment:
No matter how ugly, I think we tend to have things like queue_test_job as upper case.

Reply:
May we try to avoid the term 'job', as it is used both for a realization and a forward model in ERT?

Reply:
Change to:

> we'll also need a forward model config file, `QUEUE_TEST`

Seem reasonable?